server/src/com/thoughtworks/go/server/scheduling/BuildCauseProducerService.java

```diff
@@ -126,14 +129,14 @@ public void autoSchedulePipeline(String pipelineName, ServerHealthStateOperation
         }
     }
 
-    public void manualSchedulePipeline(Username username, PipelineConfig pipelineConfig, ScheduleOptions scheduleOptions, OperationResult result) {
-        long trackingId = schedulingPerformanceLogger.manualSchedulePipelineStart(CaseInsensitiveString.str(pipelineConfig.name()));
+    public void manualSchedulePipeline(Username username, String pipelineName, ScheduleOptions scheduleOptions, OperationResult result) {
```

Comment:
could you change `String pipelineName` to `CaseInsensitiveString pipelineName` instead as WaitForPipelineMaterialUpdate seems to use CaseInsensitiveString anyhow.